gruntfile.js

```diff
@@ -277,6 +285,8 @@ module.exports = function (grunt) {
   grunt.registerTask('test', ['env:test', 'lint', 'mkdir:upload', 'copy:localConfig', 'server', 'mochaTest', 'karma:unit']);
   grunt.registerTask('test:server', ['env:test', 'lint', 'server', 'mochaTest']);
   grunt.registerTask('test:client', ['env:test', 'lint', 'server', 'karma:unit']);
```

Comment:
I don't think e2e should be included in the *client* task. There's a clear separation between these types of tests from the others. Demonstrated by the separation of the tests in their respective folders; */client*, */server*, and */e2e* 

E2E tests behave much differently then the others, so they probably should be isolated. Adding them to the main `test` task is a good idea though, so that we can run all tests at once.